client.go

```diff
@@ -618,22 +618,23 @@ func (c *Client) GetReadyMessagesCount(queueName string) (msgCount int, err erro
 	return
 }
 
-// ConsumeReadyMessages fetches given number of messages from queue
-func (c *Client) ConsumeReadyMessages(queueName string, numberOfMessages int) (msgs []amqp.Delivery, err error) {
+// GetReadyMessages fetches given number of messages from queue
+// Note : close() should be called once all amqp.Delivery are processed
+func (c *Client) GetReadyMessages(queueName string, n int) (m []amqp.Delivery, close func() error, err error) {
```

Comment:
`Client` is designed to work with a one queue/exchange, thus you should not specify the queueName in the method args, because it is already in the `Client` fields

Reply:
Agreed. will update both methods